docs/csharp/whats-new/csharp-11.md

```diff
@@ -87,7 +87,14 @@ These types aren't directly represented in metadata. They include annotations th
 - `string` instead of `string?`.
 - `ValueTuple<int, int>` instead of `(int X, int Y)`.
 
-## Static abstract members in interfaces
+## Generic math support
+
+There are several language features that enable generic math support:
+
+- static abstract members in interfaces
```

Comment:
Should this read `static virtual members in interfaces` now that it can be `abstract` (no body) or `virtual` (a body and therefore default implementation is provided)